public/dependencies/register_assistant.tsx

```diff
@@ -3,61 +3,21 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { merge } from 'lodash';
 import React from 'react';
-import { toMountPoint } from '../../../../src/plugins/opensearch_dashboards_react/public';
-import { SavedVisualization } from '../../common/types/explorer';
-import { SavedObjectVisualization } from '../components/visualizations/saved_object_visualization';
-import { coreRefs } from '../framework/core_refs';
-import { AssistantSetup } from '../types';
-import { PPLVisualizationModal } from './components/ppl_visualization_model';
+import { Provider } from 'react-redux';
+import { AssistantSetup, IMessage } from '../types';
+import { store } from '../framework/redux/store';
+import { DataGridContainer } from './components/data_grid_container';
 
 export const registerAsssitantDependencies = (setup?: AssistantSetup) => {
   if (!setup) return;
 
-  setup.registerContentRenderer('ppl_visualization', (content) => {
```

Comment:
we don't need ppl based visualizations in chat anymore?

Reply:
No, only data grid is needed.